app/controllers/api/v2/template_combinations_controller.rb

```diff
@@ -71,6 +72,16 @@ def destroy
       def allowed_nested_id
         %w(environment_id hostgroup_id provisioning_template_id)
       end
+
+      def deprecation_warning
+        if request.path =~ /^\/environments\//
+          if params[:action] == :index
+            Foreman::Deprecation.api_deprecation_warning('This endpoint will be extracted to ForemanPuppetEnc plugin')
```

Comment:
This morning you mentioned you may want to call it `foreman_puppet`. Has that changed?

Reply:
No, it has not :(